traitsui/qt4/list_editor.py

```diff
@@ -526,7 +526,8 @@ def update_editor(self):
 
             # Remember the page for later deletion processing:
             self._uis.append([ui.control, ui, view_object, monitoring])
-
+            if len(self._uis) == 1:
+                self._tab_activated(0)
         if self.selected:
```

Comment:
Can `self._tab_activated` be placed outside the for loop? That makes it clearer that we always want the first tab to be selected. 

The tests seem do seem to pass if it's outside.

```suggestion
        self._tab_activated(0)
        if self.selected:
```

Reply:
I think this makes sense / is safe 👍